### DISCUSSION TITLE
Pagination records colors

### DISCUSSION THREAD

**Discussion by bastiengrignon** [icflorescu/mantine-datatable#258] [Q&A]
How can I change the color of the **records per page** button color. I've managned to change the color of the pagination button number (1, 2, ...) thanks to the [additional-styling](https://icflorescu.github.io/mantine-datatable/examples/additional-styling) section.
But I can't change the `backgroundColor` of the recordsPerPage menu.

Do you have an idea to do it ? 

It could be nice to have a property like `paginationColor` to be able to change it easily and everywhere, or something else more accessible :thinking:

**Comment by icflorescu** [comment]
This is actually a bug. There is a property called `paginationColor` that accepts a `MantineColor`, but as it turns out, it's not passed down to the page size selector.
Will come up with a fix in a few minutes.

Thans for noticing!

**Reply by icflorescu** [reply]
See #261.

**Comment by icflorescu** [comment]
This will be fixed in a few minutes, as soon as `v2.4.4` hits the npm registry.

**Reply by bastiengrignon** [reply]
Thanks for the quick fix :wink:

**Reply by icflorescu** [reply]
You're welcomed. I just happened to be around and available :-)
